### ISSUE TITLE
WPC和WPC2.0训练参数

### ISSUE THREAD

**Issue by ghost** [zzc-1998/MM-PCQA#3]
您好，这个模型训练过程中，根据代码中提供的参数可以在SJTU数据集上训练的很好，但是WPC和WPC2.0数据集上一直得到的结果较差（SROCC 0.49)，可以分享一下当时您训练这两个数据集设定的参数嘛

**Comment by zzc-1998** [comment]
你好，请参考repo内的train.sh的参数配置。

具体如下：

CUDA_VISIBLE_DEVICES=0 python -u train.py \
--learning_rate 0.00005 \
--model MM_PCQA \
--batch_size  8 \
--database SJTU  \
--data_dir_2d path_to_sjtu_projections/ \
--data_dir_pc path_to_sjtu_patch_2048/ \
--loss l2rank \
--num_epochs 50 \
--k_fold_num 9 \
>> logs/sjtu_mmpcqa.log


CUDA_VISIBLE_DEVICES=0 python -u train.py \
--learning_rate 0.00005 \
--model MM_PCQA \
--batch_size  8 \
--database WPC  \
--data_dir_2d path_to_wpc_projections/ \
--data_dir_pc path_to_wpc_patch_2048/ \
--loss l2rank \
--num_epochs 50 \
--k_fold_num 5 \
>> logs/wpc_mmpcqa.log

CUDA_VISIBLE_DEVICES=0 python -u train.py \
--learning_rate 0.00005 \
--model MM_PCQA \
--batch_size  8 \
--database WPC2.0  \
--data_dir_2d path_to_wpc2.0_projections/ \
--data_dir_pc path_to_wpc2.0_patch_2048/ \
--loss l2rank \
--num_epochs 50 \
--k_fold_num 4 \
>> logs/wpc2.0_mmpcqa.log

我这里可以提供WPC的训练log供你参考：
*************************************************************************************************************************
load the pretrained model, done！
Using model: MM-PCQA
Using l2rank loss
Using Adam optimizer, initial learning rate: 5e-05
Ready to train network
*************************************************************************************************************************
Epoch 1 averaged training loss: 911.0889
The current learning rate is 0.000050
Epoch 1 training time cost: 66.6278 seconds
Test results: SROCC=0.7021, KROCC=0.5256, PLCC=0.7007, RMSE=16.6320
Update best model using best_val_criterion 
Update the best Test results: SROCC=0.7021, KROCC=0.5256, PLCC=0.7007, RMSE=16.6320
Epoch 2 averaged training loss: 291.5356
The current learning rate is 0.000050
Epoch 2 training time cost: 66.6125 seconds
Test results: SROCC=0.7268, KROCC=0.5470, PLCC=0.7327, RMSE=15.8665
Update best model using best_val_criterion 
Update the best Test results: SROCC=0.7268, KROCC=0.5470, PLCC=0.7327, RMSE=15.8665
Epoch 3 averaged training loss: 249.2855
The current learning rate is 0.000050
Epoch 3 training time cost: 66.6923 seconds
Test results: SROCC=0.7523, KROCC=0.5593, PLCC=0.6807, RMSE=17.0792
Update best model using best_val_criterion 
Update the best Test results: SROCC=0.7523, KROCC=0.5593, PLCC=0.6807, RMSE=17.0792
Epoch 4 averaged training loss: 228.0509
The current learning rate is 0.000050
Epoch 4 training time cost: 66.7481 seconds
Test results: SROCC=0.7922, KROCC=0.6071, PLCC=0.7982, RMSE=14.0444
Update best model using best_val_criterion 
Update the best Test results: SROCC=0.7922, KROCC=0.6071, PLCC=0.7982, RMSE=14.0444
Epoch 5 averaged training loss: 239.8473
The current learning rate is 0.000050
Epoch 5 training time cost: 66.9706 seconds
Test results: SROCC=0.7270, KROCC=0.5273, PLCC=0.7362, RMSE=15.7769
Epoch 6 averaged training loss: 233.9770
The current learning rate is 0.000050
Epoch 6 training time cost: 66.7346 seconds
Test results: SROCC=0.7945, KROCC=0.6091, PLCC=0.7928, RMSE=14.2077
Update best model using best_val_criterion 
Update the best Test results: SROCC=0.7945, KROCC=0.6091, PLCC=0.7928, RMSE=14.2077
Epoch 7 averaged training loss: 187.2974
The current learning rate is 0.000050
Epoch 7 training time cost: 66.5150 seconds
Test results: SROCC=0.8089, KROCC=0.6194, PLCC=0.8022, RMSE=13.9183
Update best model using best_val_criterion 
Update the best Test results: SROCC=0.8089, KROCC=0.6194, PLCC=0.8022, RMSE=13.9183
Epoch 8 averaged training loss: 170.0297
The current learning rate is 0.000045
Epoch 8 training time cost: 66.9894 seconds
Test results: SROCC=0.7255, KROCC=0.5383, PLCC=0.7194, RMSE=16.1920
Epoch 9 averaged training loss: 173.7674
The current learning rate is 0.000045
Epoch 9 training time cost: 66.8328 seconds
Test results: SROCC=0.8036, KROCC=0.6075, PLCC=0.8009, RMSE=13.9586
Epoch 10 averaged training loss: 184.6821
The current learning rate is 0.000045
Epoch 10 training time cost: 66.5761 seconds
Test results: SROCC=0.7716, KROCC=0.5825, PLCC=0.7786, RMSE=14.6281
Epoch 11 averaged training loss: 192.9524
The current learning rate is 0.000045
Epoch 11 training time cost: 66.6049 seconds
Test results: SROCC=0.8359, KROCC=0.6549, PLCC=0.8369, RMSE=12.7607
Update best model using best_val_criterion 
Update the best Test results: SROCC=0.8359, KROCC=0.6549, PLCC=0.8369, RMSE=12.7607
Epoch 12 averaged training loss: 192.2851
The current learning rate is 0.000045
Epoch 12 training time cost: 67.1979 seconds
Test results: SROCC=0.7492, KROCC=0.5609, PLCC=0.7596, RMSE=15.1623
Epoch 13 averaged training loss: 178.3933
The current learning rate is 0.000045
Epoch 13 training time cost: 66.6491 seconds
Test results: SROCC=0.8248, KROCC=0.6325, PLCC=0.8259, RMSE=13.1455
Epoch 14 averaged training loss: 192.9511
The current learning rate is 0.000045
Epoch 14 training time cost: 67.0698 seconds
Test results: SROCC=0.7658, KROCC=0.5764, PLCC=0.7754, RMSE=14.7205
Epoch 15 averaged training loss: 167.6623
The current learning rate is 0.000045
Epoch 15 training time cost: 66.6250 seconds
Test results: SROCC=0.7778, KROCC=0.5830, PLCC=0.7744, RMSE=14.7504
Epoch 16 averaged training loss: 160.3173
The current learning rate is 0.000041
Epoch 16 training time cost: 67.1622 seconds
Test results: SROCC=0.7971, KROCC=0.5994, PLCC=0.8032, RMSE=13.8869
Epoch 17 averaged training loss: 171.5947
The current learning rate is 0.000041
Epoch 17 training time cost: 67.0610 seconds
Test results: SROCC=0.8219, KROCC=0.6292, PLCC=0.8225, RMSE=13.2580
Epoch 18 averaged training loss: 173.9903
The current learning rate is 0.000041
Epoch 18 training time cost: 66.6623 seconds
Test results: SROCC=0.8238, KROCC=0.6303, PLCC=0.8367, RMSE=12.7681
Epoch 19 averaged training loss: 159.9130
The current learning rate is 0.000041
Epoch 19 training time cost: 67.3795 seconds
Test results: SROCC=0.7943, KROCC=0.5968, PLCC=0.7918, RMSE=14.2405
Epoch 20 averaged training loss: 171.1829
The current learning rate is 0.000041
Epoch 20 training time cost: 67.0669 seconds
Test results: SROCC=0.8470, KROCC=0.6654, PLCC=0.8480, RMSE=12.3568
Update best model using best_val_criterion 
Update the best Test results: SROCC=0.8470, KROCC=0.6654, PLCC=0.8480, RMSE=12.3568
Epoch 21 averaged training loss: 170.1593
The current learning rate is 0.000041
Epoch 21 training time cost: 66.8068 seconds
Test results: SROCC=0.8005, KROCC=0.6227, PLCC=0.8058, RMSE=13.8051
Epoch 22 averaged training loss: 161.5014
The current learning rate is 0.000041
Epoch 22 training time cost: 66.7364 seconds
Test results: SROCC=0.7940, KROCC=0.5979, PLCC=0.7928, RMSE=14.2085
Epoch 23 averaged training loss: 174.9371
The current learning rate is 0.000041
Epoch 23 training time cost: 67.0592 seconds
Test results: SROCC=0.7782, KROCC=0.5865, PLCC=0.7869, RMSE=14.3860
Epoch 24 averaged training loss: 137.6525
The current learning rate is 0.000036
Epoch 24 training time cost: 66.7049 seconds
Test results: SROCC=0.8457, KROCC=0.6634, PLCC=0.8434, RMSE=12.5262
Epoch 25 averaged training loss: 148.3116
The current learning rate is 0.000036
Epoch 25 training time cost: 66.5353 seconds
Test results: SROCC=0.7833, KROCC=0.6029, PLCC=0.7972, RMSE=14.0756
Epoch 26 averaged training loss: 190.7232
The current learning rate is 0.000036
Epoch 26 training time cost: 66.7023 seconds
Test results: SROCC=0.8636, KROCC=0.6760, PLCC=0.8697, RMSE=11.5069
Update best model using best_val_criterion 
Update the best Test results: SROCC=0.8636, KROCC=0.6760, PLCC=0.8697, RMSE=11.5069
Epoch 27 averaged training loss: 140.3431
The current learning rate is 0.000036
Epoch 27 training time cost: 66.7309 seconds
Test results: SROCC=0.8423, KROCC=0.6494, PLCC=0.8445, RMSE=12.4836
Epoch 28 averaged training loss: 137.8987
The current learning rate is 0.000036
Epoch 28 training time cost: 67.3802 seconds
Test results: SROCC=0.8213, KROCC=0.6352, PLCC=0.8197, RMSE=13.3550
Epoch 29 averaged training loss: 123.6086
The current learning rate is 0.000036
Epoch 29 training time cost: 66.9521 seconds
Test results: SROCC=0.8210, KROCC=0.6380, PLCC=0.8225, RMSE=13.2596
Epoch 30 averaged training loss: 132.0040
The current learning rate is 0.000036
Epoch 30 training time cost: 66.4477 seconds
Test results: SROCC=0.8530, KROCC=0.6657, PLCC=0.8559, RMSE=12.0553
Epoch 31 averaged training loss: 139.0212
The current learning rate is 0.000036
Epoch 31 training time cost: 66.8905 seconds
Test results: SROCC=0.8216, KROCC=0.6314, PLCC=0.8247, RMSE=13.1849
Epoch 32 averaged training loss: 156.2805
The current learning rate is 0.000033
Epoch 32 training time cost: 66.9564 seconds
Test results: SROCC=0.8371, KROCC=0.6505, PLCC=0.8420, RMSE=12.5757
Epoch 33 averaged training loss: 134.1225
The current learning rate is 0.000033
Epoch 33 training time cost: 67.1574 seconds
Test results: SROCC=0.7887, KROCC=0.6089, PLCC=0.7962, RMSE=14.1041
Epoch 34 averaged training loss: 129.2047
The current learning rate is 0.000033
Epoch 34 training time cost: 67.1867 seconds
Test results: SROCC=0.8232, KROCC=0.6415, PLCC=0.8273, RMSE=13.0963
Epoch 35 averaged training loss: 123.3054
The current learning rate is 0.000033
Epoch 35 training time cost: 66.6079 seconds
Test results: SROCC=0.8480, KROCC=0.6588, PLCC=0.8525, RMSE=12.1871
Epoch 36 averaged training loss: 128.1395
The current learning rate is 0.000033
Epoch 36 training time cost: 67.1524 seconds
Test results: SROCC=0.8293, KROCC=0.6509, PLCC=0.8352, RMSE=12.8207
Epoch 37 averaged training loss: 122.1962
The current learning rate is 0.000033
Epoch 37 training time cost: 66.9688 seconds
Test results: SROCC=0.8542, KROCC=0.6828, PLCC=0.8614, RMSE=11.8403
Epoch 38 averaged training loss: 112.3159
The current learning rate is 0.000033
Epoch 38 training time cost: 66.4609 seconds
Test results: SROCC=0.8501, KROCC=0.6727, PLCC=0.8504, RMSE=12.2676
Epoch 39 averaged training loss: 121.3407
The current learning rate is 0.000033
Epoch 39 training time cost: 66.8107 seconds
Test results: SROCC=0.8391, KROCC=0.6481, PLCC=0.8322, RMSE=12.9270
Epoch 40 averaged training loss: 121.6807
The current learning rate is 0.000030
Epoch 40 training time cost: 66.9426 seconds
Test results: SROCC=0.8430, KROCC=0.6591, PLCC=0.8376, RMSE=12.7356
Epoch 41 averaged training loss: 127.3399
The current learning rate is 0.000030
Epoch 41 training time cost: 66.4336 seconds
Test results: SROCC=0.7146, KROCC=0.5541, PLCC=0.7230, RMSE=16.1066
Epoch 42 averaged training loss: 129.5199
The current learning rate is 0.000030
Epoch 42 training time cost: 66.7883 seconds
Test results: SROCC=0.8156, KROCC=0.6190, PLCC=0.8215, RMSE=13.2940
Epoch 43 averaged training loss: 134.2851
The current learning rate is 0.000030
Epoch 43 training time cost: 66.7571 seconds
Test results: SROCC=0.8597, KROCC=0.6823, PLCC=0.8629, RMSE=11.7816
Epoch 44 averaged training loss: 120.7997
The current learning rate is 0.000030
Epoch 44 training time cost: 67.1869 seconds
Test results: SROCC=0.8437, KROCC=0.6637, PLCC=0.8511, RMSE=12.2390
Epoch 45 averaged training loss: 106.8727
The current learning rate is 0.000030
Epoch 45 training time cost: 67.0713 seconds
Test results: SROCC=0.8436, KROCC=0.6578, PLCC=0.8472, RMSE=12.3848
Epoch 46 averaged training loss: 108.2125
The current learning rate is 0.000030
Epoch 46 training time cost: 67.3196 seconds
Test results: SROCC=0.8000, KROCC=0.6018, PLCC=0.8073, RMSE=13.7565
Epoch 47 averaged training loss: 126.2806
The current learning rate is 0.000030
Epoch 47 training time cost: 67.5054 seconds
Test results: SROCC=0.8154, KROCC=0.6288, PLCC=0.8176, RMSE=13.4237
Epoch 48 averaged training loss: 115.5771
The current learning rate is 0.000027
Epoch 48 training time cost: 67.2151 seconds
Test results: SROCC=0.8221, KROCC=0.6312, PLCC=0.8202, RMSE=13.3359
Epoch 49 averaged training loss: 96.0818
The current learning rate is 0.000027
Epoch 49 training time cost: 67.3727 seconds
Test results: SROCC=0.8270, KROCC=0.6420, PLCC=0.8308, RMSE=12.9743
Epoch 50 averaged training loss: 90.8436
The current learning rate is 0.000027
Epoch 50 training time cost: 66.6264 seconds
Test results: SROCC=0.8439, KROCC=0.6634, PLCC=0.8479, RMSE=12.3596

**Comment by ghost** [comment]
非常感谢
